spec/models/story_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Story, type: :model do
+  describe '#tagged_stories' do
+    it 'should return one story' do
+      FactoryGirl.create(:story, tags: ['NDC'])
```

Comment:
to make this a more reliable test I'd create a second story without the tag, this way we can ensure that it's only retrieving the tagged one.